hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: minor
+
+Enable :hypothesis:`hypothesis.extra.ghostwriter` to accept a :python:`@classmethod` or :python:`@staticmethod` as input.
+:hypothesis:`hypothesis.extra.ghostwriter.magic` can discover :python:`@classmethod` or :python:`@staticmethod` of classes through accepting a module or a class as input.
```

Comment:
Not quite the right syntax - see https://hypothesis--3354.org.readthedocs.build/en/3354/changes.html#current-pull-request for the preview (via "successful checks -> details" on the PR); and https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html for details (the `py` domain is enabled by default, and you probably want to use `:doc:` rather than `:module:` for the Ghostwriter - check how earlier changelog entries do it).